src/index.js

```diff
@@ -3,7 +3,11 @@ import {getQueriesForElement, prettyDOM} from 'dom-testing-library'
 export * from 'dom-testing-library'
 const mountedContainers = new Set()
 export const render = (Component, options) => {
-  const target = document.body.appendChild(document.createElement('div'))
+  let target = document.body.appendChild(document.createElement('div'))
+
+  if (options && options.target) {
```

Comment:
`options` can be `undefined`, based on examples like:

```
const {debug} = render(App)
```

i can update the syntax to use default params or something more "modern" if needed

Reply:
Yes please use a default param if you doing mind